17 Custom Middleware/src/middleware/uiNotificationMware.ts

```diff
@@ -0,0 +1,19 @@
+import * as toastr from 'toastr';
+import {UINotificationInfo} from './uiNotificationInfo'
+
+function createUINotificationMiddleware() {
```

Comment:
Just covering a simple case, in a real app we should think about how far we should get and cover notification cases using this middleware.